### PR TITLE
examples: add remote-install example

### PR DESCRIPTION
**Pull request by mrVanDalo**
@Lassulus your script was that good, that I decided to put an example in the nixos-generators project. Documentation could be better, I might improve it in the next hours.

### CODE REVIEW

File: examples/remote-install/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# remote installation iso
+
```
Comment (zimbatm) [inline examples/remote-install/README.md]: maybe add a sentence or two about the motivation?

**Comment by Lassulus** [comment]
uhm, maybe we should open a separate repository for this? I think the tor backconnect has no real connection to the nixos-generators.

**Comment by mrVanDalo** [comment]
Another repository is fine by me, but I can't create one in the nix-community repository. nix-community/nixos-generator-examples would be a good name.

**Comment by zimbatm** [comment]
I don't think it hurts too much to have a few examples added to the repo, but they should be documented. Add an example section to the top-level README.md and each example should have a README.md that explains what the example is about and how to use it. I think this is the minimum bar for them to be used.

**Comment by mrVanDalo** [comment]
@zimbatm done, here you can see the rendered version : https://github.com/mrVanDalo/nixos-generators/tree/feature/example/remote-install

**Comment by mrVanDalo** [comment]
Because @Lassulus made me do it.

**Pull request closed without merging**